Week3/normalization exercise/README.md

```diff
@@ -0,0 +1,12 @@
+# Exercise 1 : Normalization
+
+This exercise has questions like:
+
+- What columns violate 1NF?
+- Create a 3NF compliant solution.
+
+![Can you normalize this data? Said no one ever](https://i.imgflip.com/4ta2um.jpg)

```

Comment:
I am not sure if this meme adds something.
It actually makes the file less readable.